Tasks/DropValidatorV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,17 @@
+{
+  "loc.friendlyName": "Drop validator",
+  "loc.helpMarkDown": "",
+  "loc.description": "Validates a given manifest with a manifest file to verify its authenticity.",
+  "loc.instanceNameFormat": "Drop Validator $(definition)",
```

Comment:
In case we want to rename the project, I think we should use a variable to store the name so we can just replace the value of variable and have everything renamed. Also, this will help with inconsistent casing of the name: Drop Validator, drop validator, DropValidator, etc.

Reply:
The value can't be a variable, we will have to change all occurrences if we change the name in the future. We will also have to create a new task if the name has to be changed from DropValidator to something else.